mcs/class/corlib/System.Threading/HostExecutionContext.cs

```diff
@@ -52,5 +52,15 @@ public virtual HostExecutionContext CreateCopy ()
 			get { return _state; }
 			set { _state = value; }
 		}
+
+		public void Dispose ()
+		{
+			Dispose (true);
+			GC.SuppressFinalize (this);
```

Comment:
@marek-safar right, I now remember that I just copied this from https://github.com/Microsoft/referencesource/blob/master/mscorlib/system/threading/hostexecutioncontextmanager.cs#L88-92